tests/test_other.py

```diff
@@ -2346,18 +2346,26 @@ def test_embind_closure_no_dynamic_execution(self):
     create_file('post.js', '''
       Module['onRuntimeInitialized'] = function() {
         out(Module['foo'](10));
+        out(Module['bar']());
       };
     ''')
     create_file('main.cpp', r'''
+      #include <string>
+      #include <emscripten/console.h>
       #include <emscripten/bind.h>
       int foo(int x) { return x; }
+      void bar() {
+        emscripten::val(123).call<std::string>("toString");
+        emscripten_console_log("ok");
+      }
       EMSCRIPTEN_BINDINGS(baz) {
         emscripten::function("foo", &foo);
+        emscripten::function("bar", &bar);
```

Comment:
Sees like you don't even need to declare bar or bind it .. you could just do:

```
int main() {
  emscripten::val(123).call<std::string>("toString");
  emscripten_console_log("ok");
}
```

No a big deal though..